eng/common/scripts/Helpers/DocMs-Metadata-Helper.ps1

```diff
@@ -0,0 +1,55 @@
+function Generate-AadToken ($TenantId, $ClientId, $ClientSecret) {
```

Comment:
Rename "Metadata-Helpers.ps1". @danieljurek this can become the place we add other such metadata helpers.